src/features/home/pages/Home.tsx

```diff
@@ -32,7 +32,12 @@ const statusBarHeight = getStatusBarHeight(true)
 export const Home: FunctionComponent = function () {
   const navigation = useNavigation<UseNavigationType>()
   const { params } = useRoute<UseRouteType<'Home'>>()
-  const { data: userInfos } = useUserProfileInfo()
+  const { data: userInfos, refetch } = useUserProfileInfo()
+  useFocusEffect(
```

Comment:
`useFocusEffect(refetch)`

Reply:
C'est en faisant ça que j'ai l'erreur dont je te parlais sur slack : `It looks like you wrote 'useFocusEffect(async () => ...)' or returned a Promise. Instead, write the async function inside your effect and call it immediately...`